src/middlewares/authCheck.js

```diff
@@ -0,0 +1,12 @@
+const jwt = require("jsonwebtoken");
+
+module.exports = authCheck = (req, res, next) => {
+    const token =req.headers.authorization
+    const newToken = token.slice(7,token.length)
```

Comment:
if `authorization` header doesn't exist, it looks like your server will crash

Reply:
also slicing the token is weird im guessing it's something to do with `ZoIiwiY`?